### PR TITLE
Update to latest pindexer schema

### PR DESCRIPTION
**Pull request by grod220**
Closes https://github.com/penumbra-zone/dex-explorer/issues/132

Mainnet pindexer support TBD.

### CODE REVIEW

**Review by VanishMax — COMMENTED** [review]
suggestion: add the generation command to `package.json`, so it can be easily repeated by any DEX developer. You would probably need to add `kysely-codegen` to dev dependencies

File: src/shared/database/schema.ts [hunk 1]
```diff
@@ -20,43 +20,71 @@ export type JsonValue = JsonArray | JsonObject | JsonPrimitive;
...
-  id: Generated<number>;
-  low: number;
-  open: number;
-  swap_volume: number;
-}
-
 export const durationWindows = ['1m', '15m', '1h', '4h', '1d', '1w', '1mo'] as const;
 export type DurationWindow = (typeof durationWindows)[number];
 export const isDurationWindow = (str: string): str is DurationWindow =>
   durationWindows.includes(str as DurationWindow);
```
Comment (VanishMax) [inline src/shared/database/schema.ts:26]: question: are these constants and functions also generated by kysely? If not, can you move them somewhere else (e.g. `src/shared/utils`)?

File: src/shared/database/schema.ts [hunk 2]
```diff
@@ -7,56 +13,259 @@ export type Generated<T> =
...
+  supply_validators: SupplyValidators;
 }
+
+export type DB = Pick<
+  RawDB,
+  | 'dex_ex_aggregate_summary'
+  | 'dex_ex_pairs_block_snapshot'
+  | 'dex_ex_pairs_summary'
+  | 'dex_ex_price_charts'
+>;
```
Comment (grod220) [inline src/shared/database/schema.ts:271]: Previously relevant: https://github.com/penumbra-zone/dex-explorer/pull/123#pullrequestreview-2410934582 cc @cronokirby

We can't both have an auto schema generator and not include some irrelevant types. Hoping with `scripts/generate-pindexer-schema.ts` and this `Pick` it is a fair compromise given developers will not be able to query outside this list.

**Review by VanishMax — APPROVED** [review]
praise for the script covering all cases 👍

**Pull request merged**